Projects/Features/Reservation/Sources/Reservation/ReservationViewModel.swift

```diff
@@ -91,7 +91,8 @@ final class ReservationViewModel: Reducerable {
             }
         case let .showImageViewer(index):
             Task { @MainActor in
-                state.sharedImages = await fetchImages(urlStrings: state.reservation.images)
+                guard let images = state.reservation?.images else { return }
```

Comment:
이런건 Task실행 이전에해주면 좋다고생각하는편
Task자체를 실행안시키도록
```suggestion
guard let images = state.reservation?.images else { return }
Task { @MainActor in
   ...
}
```

Reply:
수정하겠습니다~